ssms/config/_modelconfig/utils.py

```diff
@@ -0,0 +1,48 @@
+"""
+Utility functions for model configuration.
+"""
+
+
+def _new_param(default: float, lower: float, upper: float) -> dict:
```

Comment:
could you make a `follow_up` issue that asks to re-evaluate and then apply this pattern throughout? 

not against this, but it's now applied inconsistently.